Chinese_Version/ch_1_Introduction/README.md

```diff
@@ -0,0 +1,54 @@
+# 第一章 简介
+
+## 什么是 BigDL-LLM
+[BigDL-LLM](https://github.com/intel-analytics/BigDL/tree/main/python/llm) 是一个库，可使 LLM（大型语言模型）在低成本 PC 上快速[^1]运行（无需独立显卡）。它作为开源项目 [BigDL](https://github.com/intel-analytics/bigdl) 的一部分发布，采用 Apache 2.0 许可。
+
+
+## 您能用 BigDL-LLM 做些什么
+您可以使用 BigDL-LLM 运行 _任何 [HuggingFace transformer](https://huggingface.co/docs/transformers/index) 模型_。它使用低精度技术、现代硬件加速和最新软件优化自动优化和加速 LLM。
+
+只需修改一行代码，基于 HuggingFace transformer 的应用程序就能在 BigDL-LLM 上运行，而且您会立即观察到显著的速度提升[^1]。
+
+```python
+# 修改 import，在加载模型的时候指定精度
+from bigdl.llm.transformers import AutoModelForCausalLM
+model = AutoModelForCausalLM.from_pretrained('/path/to/model/', load_in_4bit=True)
+# 模型推理部分的代码无需修改
+...
+```
+
+BigDL-LLM 提供各种低精度优化（如 INT4/INT5/INT8），允许您在资源有限的 PC 上运行 LLM。例如，您可以在 16G 内存的笔记本电脑上运行 7B 或 13B 模型，而且延迟非常低[^1]。
+
+#### 在英特尔 12 代酷睿电脑上运行 7B 模型（实时屏幕画面）:
+
+<p align="left">
```

Comment:
Please change the link accordingly. Refer to https://github.com/intel-analytics/bigdl-llm-tutorial/pull/34/files

Also change 7B -> 6B here in title

Reply:
Done.